### ISSUE TITLE
addon's public assets are not copied under addon's namespace to dummy app for testing

### ISSUE THREAD

**Issue by mattmarcum** [ember-cli/ember-cli#2358]
It looks like the behavior of an addon's `public` tree doesn't match up between the dummy app and a real word consuming app.  For an addon I'm developing the public assets get copied over to the root of the dummy app, but when I test the addon in a different consuming app the public assets get copied to a folder named after the addon - which is the expected behavior.

I haven't tracked this down yet, just curious if anyone else has seen this behavior?

**Comment by stefanpenner** [comment]
just tested this, it appears to work as expected.

```
my-cool-addon/public/index.html (containing <h1> foo</h1>)
ember s
curl http://0.0.0.0:4200/my-cool-addon/index.html (results in <h1> foo</h1)
```

**Comment by jayphelps** [comment]
@stefanpenner I didn't see this ticket at the time, but it was fixed by #2433

**Comment by stefanpenner** [comment]
@jayphelps :+1:
